runtime/compiler/env/j9method.cpp

```diff
@@ -2787,6 +2787,7 @@ void TR_ResolvedJ9Method::construct()
       {x(TR::java_lang_StringCoding_decode, "decode", "(Ljava/nio/charset/Charset;[BII)[C")},
       {x(TR::java_lang_StringCoding_encode, "encode", "(Ljava/nio/charset/Charset;[CII)[B")},
       {x(TR::java_lang_StringCoding_implEncodeISOArray, "implEncodeISOArray", "([BI[BII)I")},
+      {x(TR::java_lang_StringCoding_implEncodeISOArray, "implEncodeAsciiArray", "([CI[BII)I")},
```

Comment:
Typo?  I think this should be `TR::java_lang_StringCoding_implEncodeAsciiArray` rather than `TR::java_lang_StringCoding_implEncodeISOArray`

Reply:
Sorry, looks like an oversight when I was porting over the changes for this PR. It should be fixed now: https://github.com/eclipse-openj9/openj9/commit/d34659a9719664cd8c1852ef62c2f9046c53d177